### PR TITLE
@kanaabe => Gucci nav

### PR DESCRIPTION
**Pull request by eessex**
This is still somewhat in progress, but I hope to get it onto staging for some QA. 

- Makes nav links in sectionsNav functional
- Adds fixedHeader component, with drop-down sectionsNav on scroll up, hover, and load (if slug)
- Adds waypoints that set App's 'activeSection' state while scrolling
- Adds waypoint to fade in fixedHeader 

Tests will come tomorrow in a separate PR.
TODO:  Mobile drop-down nav

### CODE REVIEW

File: desktop/apps/editorial_features/components/gucci/components/partner/__tests__/partner_block.test.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import { shallow } from 'enzyme'
```
Comment (damassi) [inline desktop/apps/editorial_features/components/gucci/components/partner/__tests__/partner_block.test.js]: Yesss -- have we used Enzyme in force before? If not, great addition 👍

Comment (damassi) [reply desktop/apps/editorial_features/components/gucci/components/partner/__tests__/partner_block.test.js]: One suggestion I would make is to always try to use `mount` as it better reflects whats going on in the DOM in a real-world environment. The speed difference is negligible.

File: desktop/apps/editorial_features/components/gucci/components/section/section.jsx [hunk 2]
```diff
@@ -1,32 +1,37 @@
...
 import { SectionText } from './section_text.jsx'
 
 export const Section = (props) => {
-  const { section, curation } = props
+  const { section, onScrollOver } = props
 
   return (
     <SectionContainer>
       <SectionVideo section={section} />
+      <Waypoint onEnter={onScrollOver} />
```
Comment (kanaabe) [inline desktop/apps/editorial_features/components/gucci/components/section/section.jsx]: 👍

**Comment by kanaabe** [comment]
Very straightforward and well-tested! 🥂

**Pull request merged**